packages/typescript/internal/ts_project.bzl

```diff
@@ -206,7 +211,7 @@ def ts_project_macro(
         composite = False,
         incremental = False,
         emit_declaration_only = False,
-        tsc = _DEFAULT_TSC,
+        tsc = None,
```

Comment:
What's this change?

Reply:
that's because the macro evaluates labels in the callers workspace
so `//typescript/bin` becomes `@my_wksp//typescript/bin`
but inside the rule it's evaluated in the context of the repo declaring the rule, so it becomes @npmwhatever//typescript/bin like we want
anyway it was always silly for that macro to bother with a default since the rule already had it